src/context/MxGraphContext.ts

```diff
@@ -6,7 +6,7 @@ interface IEdge {
 }
 
 // tslint:disable-next-line: no-empty-interface
-interface IVertex {
+export interface IVertex {
```

Comment:
My last PR was merged and you need to resolve the conflicts here. I moved all the interfaces to files under types directory.